core/templates/services/UpgradedServices.ts

```diff
@@ -164,6 +164,8 @@ import { ExtensionTagAssemblerService } from
 import { ExtractImageFilenamesFromStateService } from
   // eslint-disable-next-line max-len
   'pages/exploration-player-page/services/extract-image-filenames-from-state.service';
+import { FeaturedTranslationLanguageFactory } from
```

Comment:
This needs to be added in the `oppia-angular-root.component.ts ` as well, right @srijanreddy98 ?

Please follow point number 12 under this: https://github.com/oppia/oppia/wiki/Angular-Migration#implementation-details-to-migrate-services.

Reply:
Done